log/view_log.py

```diff
@@ -0,0 +1,11 @@
+from django.http import HttpResponse
+
+def index(request):
+    logs = get_logs()
+    return HttpResponse('<br>'.join(logs))
+
+def get_logs():
+    with open('./_site/community.log') as log_file:
+        for line in log_file:
+            if 'DEBUG' not in line:
```

Comment:
no this is not how you filter the log.  `logging` and django have official mechanisms for filtering the log.

We need things like access tokens to be never stored anywhere, at all.  Leaving them lying around in files is begging for leaks to occur again due to mistakes.

Reply:
ok. I'll follow the official mechanisms